rwengine/src/ai/TrafficDirector.cpp

```diff
@@ -93,6 +94,23 @@ void TrafficDirector::setDensity(ai::NodeType type, float density) {
     }
 }
 
+uint16_t TrafficDirector::assignDriver(std::string vehiclename) {
+    enum ped_types {
+      COP = 1,
+      MEDIC = 5,
+      FIREMAN = 6
+    };
```

Comment:
I'm not sure would I move this out of the function.. But formatting should be:

```suggestion
    enum ped_types { COP = 1, MEDIC = 5, FIREMAN = 6 };
```

Reply:
I want to move it out, but I need to get the whole list of ped_types, I just filled few ones needed for this use, feels a bit hacky, but does work, so...